db/data/development-data/index.js

```diff
@@ -2,3 +2,5 @@ exports.categoryData = require('./categories.js');
 exports.commentData = require('./comments.js');
 exports.reviewData = require('./reviews.js');
 exports.userData = require('./users.js');
+
+module.exports = {categoryData, reviewData, userData, commentData}
```

Comment:
This will cause issues.  categoryData, reviewData etc. aren't defined (exports.categoryData has been defined), so export on line 6 will cause issues.

Either export them individually (leaving lines 1-4 as is, with no line 6) OR require them as variables (not on exports.) and then export them as on line 6.